t/fixtures/emma/emma/emmatest.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SubmissionPackage submission_id="u5f94f62cq42" xmlns="https://emma.lib.virginia.edu/schema" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <emma_recordId>hathiTrust-hvd.32044086819505-epub-1.1</emma_recordId>
+  <emma_titleId>648a2004-ced1-4281-988f-a113c01b9d63</emma_titleId>
+  <emma_repository>hathiTrust</emma_repository>
+  <emma_collection>
+    <xs:string>Harvard University</xs:string>
+  </emma_collection>
+  <emma_repositoryRecordId>hvd.32044086819505</emma_repositoryRecordId>
+  <emma_lastRemediationDate>2020-07-29T16:35:31Z</emma_lastRemediationDate>
```

Comment:
Could include this as the "creation" date. No record of who did the remediation. I suppose we could omit the agent.